package.json

```diff
@@ -22,9 +22,10 @@
     "moment": "^2.29.3"
   },
   "scripts": {
-    "solhint": "./node_modules/.bin/solhint -f table src/**/*.sol",
-    "prettier:solidity": "./node_modules/.bin/prettier --write src/**/*.sol",
+    "solhint": "solhint -f table 'src/**/*.sol'",
+    "prettier:solidity": "prettier --write 'src/**/*.sol'",
+    "prettier:solidity:check": "prettier --check 'src/**/*.sol'",
```

Comment:
I am not sure if this is needed since I added the prettier plugin to solhint and running `solhint` already checks for both linting and formatting issues.

Reply:
Oops, didn't see that. I'll revert